test/back_end/api_get_test.spec.js

```diff
@@ -5,9 +5,9 @@ chai.use(chaiHttp);
 const expect = chai.expect;
 
 describe("Test GET requests to API", function() {
-    it("should return user info for prof1", async function() {
+    it("should return user info for Joshua Cook", async function() {
         let fetched = await fetch(
-            "http://localhost:3000/get_info.php?what=user_info&user_id=prof1"
+            "http://localhost:3000/get_info.php?what=user_info&user_id=cook19"
         ).catch(function(err) {
```

Comment:
No mixing `await` and `.fetch()`

Reply:
I have to wait for the fetch result to come back before I can do assert statements. Do you mean removing the `.catch`?

Reply:
I don't follow. `fetch` returns a promise which you can use `.catch` with. `await` awaits a promise and turns `.catch` into a regular JS error. Thus
```
fetch(...).catch(...)
```
is the same as
```
try {
   await fetch(...)
} catch (e) {
   ...
}
```

Reply:
That what I meant, you said to remove the `.fetch` which I got me thinking you wanted to not use `await` . I'll update according to that.